### PR TITLE
Add Markdoc page to website marketing

### PR DESCRIPTION
**Pull request by ericholscher**
This links to our newly created doc page

----
📚 Documentation preview 📚: https://readthedocs-about--336.org.readthedocs.build/

### CODE REVIEW

File: content/pages/tools/markdoc.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+               onclick="jQuery('#signup-modal').modal('show');">
+              <i class="fad fa-play icon"></i>
+              Create an account
+            </a>
+            </p>
+          </div>
+
+          <p>
+            Learn more about using Markdoc with Read the Docs in our <a href="https://docs.readthedocs.io/en/stable/intro/markdoc.html">documentation</a>.
+          </p>
```
Comment (humitos) [inline content/pages/tools/markdoc.html:128]: This last paragraph doesn't render nicely in the page. It would be good if we find another place for it.

**Review by humitos — COMMENTED** [review]
This looks good to me.

Now, I feel that we are repeating ourselves by having a "marketing page" _and_ a "documentation page" for every tool. I think we should think a little more about this to avoid this repetition.

We should highlight _all the features_ that Read the Docs adds on top of any documentation tool here with nice copy, images and GIF examples in the marketing site with a list of all the supported doctools pointing to our specific documentation pages. This will simplify the maintenance and also show why Read the Docs is different than a regular hosting providing, promoting what we want to promote: our specific features.

What do you think?

**Comment by ericholscher** [comment]
> This looks good to me.
> 
> Now, I feel that we are repeating ourselves by having a "marketing page" _and_ a "documentation page" for every tool. I think we should think a little more about this to avoid this repetition.
> 
> [We should highlight _all the features_ that Read the Docs adds on top of any documentation tool here](https://github.com/readthedocs/website/pull/336/files#diff-ede93a789fc71b167b4781a9b3fe909d20bf1f2e1964cc54be0d4ed3e4148aceR106-R110) with nice copy, images and GIF examples in the marketing site with a list of all the supported doctools pointing to our specific documentation pages. This will simplify the maintenance and also show why Read the Docs is different than a regular hosting providing, promoting what we want to promote: our specific features.
> 
> What do you think?

Strong agree. I just did the quick version to get some SEO out the door, but I definitely think we want to make these a bit more systematic and nicer over time with stuff that you mentioned.

**Comment by ericholscher** [comment]
@humitos :+1: on merging this for now, and we can iterate on all the landing pages for style?

**Comment by ericholscher** [comment]
@humitos would be good to either :+1: this, or maybe we close this out, and work on the refactor that we mentioned?

**Comment by ericholscher** [comment]
@humitos Sounds good. We should break out a roadmap card to track that work, and put it on the next sprint perhaps?

**Pull request merged**